### PR TITLE
[RC Fix] Hide some network options until next release

### PR DESCRIPTION
**Pull request by BaronGreenback**
Some of the network options are not yet implemented. (Was hoping to have more code implemented in this release).

Have hidden the un-implemented controls.

### CODE REVIEW

File: src/controllers/dashboard/networking.html [hunk 1]
```diff
@@ -105,7 +105,7 @@ <h2 class="sectionTitle">${TabNetworking}</h2>
                             </label>
                             <div class="fieldDescription checkboxFieldDescription">${LabelEnableAutomaticPortMapHelp}</div>
                         </div>
-                        <div class="checkboxContainer checkboxContainer-withDescription">
+                        <div class="checkboxContainer checkboxContainer-withDescription" style="display:none">
```
Comment (Camc314) [inline src/controllers/dashboard/networking.html:108]:
```suggestion
                        <div class="checkboxContainer checkboxContainer-withDescription hide">
```

File: src/controllers/dashboard/networking.html [hunk 2]
```diff
@@ -141,7 +141,7 @@ <h2 class="sectionTitle">${TabNetworking}</h2>
                     </fieldset>
 
 
-                    <fieldset class='verticalSection verticalSection-extrabottompadding'>
+                    <fieldset class='verticalSection verticalSection-extrabottompadding' style="display:none">
```
Comment (Camc314) [inline src/controllers/dashboard/networking.html:144]:
```suggestion
                    <fieldset class='verticalSection verticalSection-extrabottompadding hide'>

File: src/controllers/dashboard/networking.html [hunk 3]
```diff
@@ -159,17 +159,17 @@ <h2 class="sectionTitle">${TabNetworking}</h2>
                             <input is="emby-input" type="text" id="txtPublishedServer" label="${LabelPublishedServerUri}" />
                             <div class="fieldDescription">${LabelPublishedServerUriHelp}</div>
                         </div>
-                        <div class="inputContainer">
+                        <div class="inputContainer" style="display:none">
```
Comment (Camc314) [inline src/controllers/dashboard/networking.html:162]:
```suggestion
                        <div class="inputContainer hide">

File: src/controllers/dashboard/networking.html [hunk 4]
```diff
@@ -159,17 +159,17 @@ <h2 class="sectionTitle">${TabNetworking}</h2>
...
                             <input is="emby-input" type="text" id="txtUDPPortRange" pattern="[0-9\-]*" label="${LabelUDPPortRange}"  />
                             <div class="fieldDescription">${LabelUDPPortRangeHelp}</div>
                         </div>
-                        <div class="inputContainer">
+                        <div class="inputContainer" style="display:none">
```
Comment (Camc314) [inline src/controllers/dashboard/networking.html:166]:
```suggestion
                        <div class="inputContainer hide">

File: src/controllers/dashboard/networking.html [hunk 5]
```diff
@@ -159,17 +159,17 @@ <h2 class="sectionTitle">${TabNetworking}</h2>
...
                             <input is="emby-input" type="text" id="txtHDHomerunPortRange" pattern="[0-9\-]*" label="${LabelHDHomerunPortRange}" />
                             <div class="fieldDescription">${LabelHDHomerunPortRangeHelp}</div>
                         </div>
                     </fieldset>
 
-                    <fieldset class='verticalSection verticalSection-extrabottompadding'>
+                    <fieldset class='verticalSection verticalSection-extrabottompadding' style="display:none">
```
Comment (Camc314) [inline src/controllers/dashboard/networking.html:172]:
```suggestion
                    <fieldset class='verticalSection verticalSection-extrabottompadding hide'>

**Review by Camc314 — COMMENTED** [review]
We've got a `hide` class you can use ;)

File: src/controllers/dashboard/networking.js [hunk 6]
```diff
@@ -93,6 +93,10 @@ import alert from '../../components/alert';
             return 'The http and https ports must be different.';
         }
 
+        if (!form.querySelector('#chkEnableIP6').checked && !form.querySelector('#chkEnableIP4').checked) {
+            return 'Either IPv4 or IPv6 need to be checked.'
```
Comment (thornbill) [inline src/controllers/dashboard/networking.js:97]: Lint is failing due to a missing semicolon here.

**Comment by BaronGreenback** [comment]
Thx

**Review by joshuaboniface — APPROVED** [review]

**Review by dkanada — APPROVED** [review]

**Pull request merged**